tools/msg.c

```diff
@@ -242,6 +244,7 @@ bool bl_msg_yaml_parse(FILE *file, union bl_msg_data *msg)
 	case BL_MSG_SAMPLE_DATA32:
 		msg->sample_data.channel = bl_msg__yaml_read_unsigned(file, "Channel", &ok);
 		msg->sample_data.count   = bl_msg__yaml_read_unsigned(file, "Count",   &ok);
+		msg->sample_data.src_led = bl_msg__yaml_read_unsigned(file, "Source: LED",   &ok);
```

Comment:
This needs to go above "Count", because the parser ordering needs to match the print ordering in `bl_msg_yaml_print`.

Reply:
+1